### PR TITLE
[Docs] CONTRIBUTING.rst: Update the list of maintainers

### PR DESCRIPTION
**Pull request by dimakuv**
## Description of the changes 

As discussed in the Gramine core meeting, we'd like to:
- split the list of maintainers into active and inactive,
- amend the rules slightly so that only active maintainers vote,
- add @woju as the active maintainer.

## How to test this PR? 

We need to agree and vote on these changes. The comments/reviews in this PR will serve as the voting procedure.

- - -
This change is [Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)

### CODE REVIEW

**Review by dimakuv — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: 0 of 1 files reviewed, 1 unresolved discussion, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel)

*[a discussion](https://reviewable.io/reviews/gramineproject/gramine/1095#-NKt8SEp8AEug_OJw99J:-NKt8SEq3A9Wlj2TFtOa:b5k0cr1) (no related file):*
@mkow @boryspoplawski @pwmarcz @monavij @donporter @chiache @yamahata @omeg All, please review this PR and leave your comment/approval.

___

**Review by oshogbo — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: 0 of 1 files reviewed, 2 unresolved discussions, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel) (waiting on @dimakuv)

___
*[`CONTRIBUTING.rst` line 272 at r1](https://reviewable.io/reviews/gramineproject/gramine/1095#-NKwSYjkCgh1Lth-sL3C:-NKwSYjkCgh1Lth-sL3D:bhejzk4) ([raw file](https://github.com/gramineproject/gramine/blob/e0add7de8645519ae44b5d4d293ebec064265db3/CONTRIBUTING.rst#L272)):*
> ```reStructuredText
> 
> * Paweł Marczewski (Invisible Things Lab/Intel)
> * Borys Popławski (Invisible Things Lab/Intel)
> ```

My 5 cents (as I'm not maintainer): we should give Borys more time before moving him to inactive member - at least 1 month (preferred 3 months).

**Review by donporter — APPROVED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: all files reviewed, 2 unresolved discussions, not enough approvals from maintainers (1 more required) (waiting on @oshogbo)

___
*[`CONTRIBUTING.rst` line 272 at r1](https://reviewable.io/reviews/gramineproject/gramine/1095#-NKwSYjkCgh1Lth-sL3C:-NKx1Nt86FfH691pa2Kn:b-ehgdfi) ([raw file](https://github.com/gramineproject/gramine/blob/e0add7de8645519ae44b5d4d293ebec064265db3/CONTRIBUTING.rst#L272)):*
<details><summary><i>Previously, oshogbo (Mariusz Zaborski) wrote…</i></summary><blockquote>

My 5 cents (as I'm not maintainer): we should give Borys more time before moving him to inactive member - at least 1 month (preferred 3 months).
</blockquote></details>

I'm fine with this, although my understanding is that he has signaled he does not plan to re-engage in the next 1--3 months. I am very happy to transition him back to an active maintainer if he wishes.

**Review by dimakuv — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: all files reviewed, 2 unresolved discussions, not enough approvals from maintainers (1 more required) (waiting on @oshogbo)

___
*[`CONTRIBUTING.rst` line 272 at r1](https://reviewable.io/reviews/gramineproject/gramine/1095#-NKwSYjkCgh1Lth-sL3C:-NKxO_H786VBkljK4ctI:b2s2wmu) ([raw file](https://github.com/gramineproject/gramine/blob/e0add7de8645519ae44b5d4d293ebec064265db3/CONTRIBUTING.rst#L272)):*
<details><summary><i>Previously, donporter (Don Porter) wrote…</i></summary><blockquote>

I'm fine with this, although my understanding is that he has signaled he does not plan to re-engage in the next 1--3 months. I am very happy to transition him back to an active maintainer if he wishes.
</blockquote></details>

@oshogbo Why? Borys can still review, he just won't have the voting right (we actually never used votings before this PR, so it's not that Borys will miss anything).

**Review by pwmarcz — DISMISSED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: all files reviewed, 2 unresolved discussions (waiting on @oshogbo)

**Review by mkow — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: all files reviewed, 3 unresolved discussions (waiting on @dimakuv and @oshogbo)

___
*[`CONTRIBUTING.rst` line 273 at r1](https://reviewable.io/reviews/gramineproject/gramine/1095#-NKxa5ka54m144fXp27W:-NKxa5kb8iEN7sb0w9Qy:br1kkkb) ([raw file](https://github.com/gramineproject/gramine/blob/e0add7de8645519ae44b5d4d293ebec064265db3/CONTRIBUTING.rst#L273)):*
> ```reStructuredText
> * Paweł Marczewski (Invisible Things Lab/Intel)
> * Borys Popławski (Invisible Things Lab/Intel)
> * Rafał Wojdyła (Invisible Things Lab/Golem)
> ```

I'd drop `/company` (i.e. the second one) from the inactive list - the format on the active list is "company someone's working at / company sponsoring this work" - the second part was changing in time when Gramine sponsors changed, and I think it doesn't make sense for inactive people?

**Review by dimakuv — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: 0 of 1 files reviewed, 3 unresolved discussions, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "fixup! " found in commit messages' one-liners (waiting on @donporter, @mkow, and @oshogbo)

___
*[`CONTRIBUTING.rst` line 273 at r1](https://reviewable.io/reviews/gramineproject/gramine/1095#-NKxa5ka54m144fXp27W:-NKxiGIFEfxEuyCQO5XX:b-896fix) ([raw file](https://github.com/gramineproject/gramine/blob/e0add7de8645519ae44b5d4d293ebec064265db3/CONTRIBUTING.rst#L273)):*
<details><summary><i>Previously, mkow (Michał Kowalczyk) wrote…</i></summary><blockquote>

I'd drop `/company` (i.e. the second one) from the inactive list - the format on the active list is "company someone's working at / company sponsoring this work" - the second part was changing in time when Gramine sponsors changed, and I think it doesn't make sense for inactive people?
</blockquote></details>

Done.

**Review by donporter — DISMISSED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: all files reviewed, 3 unresolved discussions, not enough approvals from maintainers (1 more required), "fixup! " found in commit messages' one-liners (waiting on @mkow and @oshogbo)

**Review by boryspoplawski — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: all files reviewed, 3 unresolved discussions, "fixup! " found in commit messages' one-liners (waiting on @mkow and @oshogbo)

**Review by monavij — DISMISSED** [review]
Changes look good to me.

**Review by mkow — COMMENTED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: all files reviewed, 2 unresolved discussions, "fixup! " found in commit messages' one-liners (waiting on @dimakuv and @donporter)

___
*[`CONTRIBUTING.rst` line 272 at r1](https://reviewable.io/reviews/gramineproject/gramine/1095#-NKwSYjkCgh1Lth-sL3C:-NLW69g78g0eRcxcvpN8:bf612m4) ([raw file](https://github.com/gramineproject/gramine/blob/e0add7de8645519ae44b5d4d293ebec064265db3/CONTRIBUTING.rst#L272)):*
<details><summary><i>Previously, oshogbo (Mariusz Zaborski) wrote…</i></summary><blockquote>

I just like the idea of "grace period" but I won't complain to much :)
</blockquote></details>

+1 to having some grace period (3 months?)

**Review by dimakuv — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: 0 of 1 files reviewed, 2 unresolved discussions, not enough approvals from maintainers (1 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "fixup! " found in commit messages' one-liners (waiting on @boryspoplawski, @donporter, and @mkow)

___
*[`CONTRIBUTING.rst` line 272 at r1](https://reviewable.io/reviews/gramineproject/gramine/1095#-NKwSYjkCgh1Lth-sL3C:-NLWNgRW6tfHQRVVyuu2:b69l42u) ([raw file](https://github.com/gramineproject/gramine/blob/e0add7de8645519ae44b5d4d293ebec064265db3/CONTRIBUTING.rst#L272)):*
<details><summary><i>Previously, mkow (Michał Kowalczyk) wrote…</i></summary><blockquote>

+1 to having some grace period (3 months?)
</blockquote></details>

Done. I am not against the grace period.

**Review by dimakuv — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: 0 of 1 files reviewed, 2 unresolved discussions, not enough approvals from maintainers (1 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "fixup! " found in commit messages' one-liners (waiting on @boryspoplawski, @donporter, and @mkow)

*[a discussion](https://reviewable.io/reviews/gramineproject/gramine/1095#-NKt8SEp8AEug_OJw99J:-NLWOYoFBF2JNcTkBl3F:b-941g4a) (no related file):*
<details><summary><i>Previously, dimakuv (Dmitrii Kuvaiskii) wrote…</i></summary><blockquote>

@mkow @boryspoplawski @pwmarcz @monavij @donporter @chiache @yamahata @omeg All, please review this PR and leave your comment/approval.
</blockquote></details>

Currently, out of 9 maintainers, we have 5 approvals:
- @dimakuv
- @boryspoplawski 
- @pwmarcz 
- @monavij 
- @donporter 

and 1 pending approval:
- @mkow 

Also, 3 maintainers didn't show up yet:
- @chiache 
- @omeg 
- @yamahata 

Therefore, out of 9 maintainers, we will have 6 approvals (pending @mkow), which is a 66% vote, enough to merge this PR (and thus change the maintainers list). As soon as @mkow agrees, I suggest to merge this. Blocking for now.

___

**Review by mkow — DISMISSED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: all files reviewed, 1 unresolved discussion, "fixup! " found in commit messages' one-liners

**Review by dimakuv — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: all files reviewed, 1 unresolved discussion, not enough approvals from maintainers (1 more required), not enough approvals from different teams (1 more required, approved so far: Intel)

*[a discussion](https://reviewable.io/reviews/gramineproject/gramine/1095#-NKt8SEp8AEug_OJw99J:-NLtxrfKAmykmWXnxaUa:bope2gp) (no related file):*
<details><summary><i>Previously, dimakuv (Dmitrii Kuvaiskii) wrote…</i></summary><blockquote>

Currently, out of 9 maintainers, we have 5 approvals:
- @dimakuv
- @boryspoplawski 
- @pwmarcz 
- @monavij 
- @donporter 

and 1 pending approval:
- @mkow 

Also, 3 maintainers didn't show up yet:
- @chiache 
- @omeg 
- @yamahata 

Therefore, out of 9 maintainers, we will have 6 approvals (pending @mkow), which is a 66% vote, enough to merge this PR (and thus change the maintainers list). As soon as @mkow agrees, I suggest to merge this. Blocking for now.
</blockquote></details>

@mkow approved, so we consider merging this PR now.

@boryspoplawski @pwmarcz @monavij @donporter @mkow Please verify the changes again -- this is the last chance to modify them.

___

**Review by mkow — APPROVED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: all files reviewed, 1 unresolved discussion (waiting on @boryspoplawski, @chiache, @donporter, @monavij, @omeg, @pwmarcz, and @yamahata)

**Review by dimakuv — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1095)* status: :shipit: complete! all files reviewed, all discussions resolved

*[a discussion](https://reviewable.io/reviews/gramineproject/gramine/1095#-NKt8SEp8AEug_OJw99J:-NLz2p588DSUS71KhxQS:b-mdp2qr) (no related file):*
<details><summary><i>Previously, dimakuv (Dmitrii Kuvaiskii) wrote…</i></summary><blockquote>

@mkow approved, so we consider merging this PR now.

@boryspoplawski @pwmarcz @monavij @donporter @mkow Please verify the changes again -- this is the last chance to modify them.
</blockquote></details>

Ok, time to merge.

___

**Pull request merged**
